lib/components/user/monitored-trip/saved-trip-screen.js

```diff
@@ -239,6 +241,7 @@ class SavedTripScreen extends Component {
               <Form noValidate>
                 <SavedTripEditor
                   {...props}
+                  hasMobilityProfile={Boolean(loggedInUser.mobilityProfile)}
```

Comment:
This needs to come from config (it might not be initially populated if users skip steps in the account setup).